node-installer/script/installer.sh

```diff
@@ -42,6 +42,8 @@ if ! grep -q spin $NODE_ROOT$CONTAINERD_CONF; then
     echo '
 [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.spin]
```

Comment:
Don't we also need to do something about dbus needing a restart?

Reply:
If there needs a restart it should be the work kwasm should do, correct me if I am wrong